Importer/Interpreter/EntityDataInterpreter.php

```diff
@@ -155,7 +156,12 @@ protected function interpretRow(array $row, $associative)
 
         $validationViolations = $this->validator->validate($entity);
         if ($validationViolations->count() > 0) {
-            throw new InvalidEntityException($validationViolations);
+            $errors = [];
+            foreach ($validationViolations as $violation) {
+                $exception = new InterpreterException($violation->getMessage());
+                $errors[] = new FieldError($exception, $this->extractFieldNameFromPropertyPath($violation->getPropertyPath()));
+            }
+            throw new InvalidLineException($errors);
```

Comment:
Does it make sense to keep the name `InvalidEntityException`?
If it does it might be better to keep this name so that we don't break BC.